src/lib.rs

```diff
@@ -377,6 +386,16 @@
 //!
 //! Advanced time zone handling is not yet supported.
 //! For now you can try the [Chrono-tz](https://github.com/chronotope/chrono-tz/) crate instead.
+//!
+//! ## Rust version requirements
+//!
+//! The Minimum Supported Rust Version (MSRV) is currently **Rust 1.56.0**.
+//!
```

Comment:
Can you please also copy the line "The MSRV is explicitly tested in CI. It may be bumped in minor releases, but this is not done lightly."

And drop the extra lines below.